src/kernel/include/shared/cos_config.h

```diff
@@ -10,7 +10,7 @@
 
 // After how many seconds should schedulers print out their information?
 #define SCHED_PRINTOUT_PERIOD  30
-//#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
+#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
```

Comment:
we want assertions on by default.  I'd love a separate PR that just makes this change to get it in ASAP.